upgrade-notes/backend_20250915_140343.md

```diff
@@ -0,0 +1,22 @@
+
+- `Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig` constructor now has new required `$baseUrl` and optional `$postfix` parameters
+    - `DomainConfig::getUrl()` method returns the whole URL including the postfix, if you need just the host part, use `DomainConfig::getBaseUrl()` method instead
+- `Shopsys\FrameworkBundle\Component\Domain\Domain::MAIN_ADMIN_DOMAIN_ID` constant has been removed as the administration domain is no longer tied to a specific domain ID
+- there is new `Shopsys\FrameworkBundle\Component\Router\AdministrationRouter` that needs to be used in the administration context instead of `CurrentDomainRouter` when generating/matching the admin routes
+    - if you need to generate a URL for a specific domain in the administration context, use `DomainRouterFactory::getRouter()` method with the specific domain ID to get a `DomainRouter` instance for that domain
+- using `Shopsys\FrameworkBundle\Component\Domain\Domain::getCurrentDomainConfig()` in the administration context now throws `NoDomainSelectedException` exception, use `Shopsys\FrameworkBundle\Component\Domain\Domain::getDomainConfigById()` instead
+- check translations of your email templates that are used for sending the emails for the administrators (e.g., 2FA, reset password)
+    - translation based on the current administrator locale is now used instead of the current domain locale
+    - `Domain::getFirstDomainIdMatchingAdminSelectedLocale()` method can be used to get a domain ID that matches the current administrator locale
+- `Shopsys\FrameworkBundle\Twig\ImageExtension::getImageUrl()` method now requires `int $domainId` parameter
+- `Shopsys\FrameworkBundle\Component\Context\AdminContext` constructor parameter `$adminRoutePrefixes` has been renamed to `$additionalAdminPathPrefixes`
+    - the meaning of the parameter has changed, it now represents additional path prefixes instead of the original route names for the administration context
+    - the configuration parameter name has changed from `admin_context_route_prefixes` to `admin_context_additional_path_prefixes`
+- `Shopsys\FrameworkBundle\Component\Router\NormalizeUrlTrailingSlashSubscriber` has been renamed to `NormalizeAdminUrlTrailingSlashSubscriber`
+    - the subscriber now uses `Shopsys\FrameworkBundle\Component\Router\AdministrationRouter $administrationRouter` instead of the original `CurrentDomainRouter $currentDomainRouter`
+- `Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\NormalizeUrlTrailingSlashListener` class has been removed as redundant
+    - the functionality is covered by `NormalizeAdminUrlTrailingSlashSubscriber`
+- `Shopsys\FrameworkBundle\Model\Sitemap\SitemapListener::addUrlForHomepage()` method no longer accepts `string $section` parameter
+    - the section name is now resolved using `SitemapFacade::getSectionNameForDomainConfig()` method
+- see #project-base-diff to update your project
```

Comment:
Should we also mention need to update deployment bundle?

Reply:
Good catch, I will upgrade the bundle once it is ready (in progress here - https://github.com/shopsys/deployment/pull/57) and explicitly mention that in the upgrade notes to upgrade it on a project